packages/pd-manager-rest-api/src/api-functions.ts

```diff
@@ -0,0 +1,94 @@
+import { checkAuth, sendErrorResponse, ISingleEndpointOpts } from '@sphereon/ssi-express-support'
+import { Request, Response, Router } from 'express'
+import { IRequiredContext } from './types'
+import { AddDefinitionItemArgs, PersistDefinitionArgs, UpdateDefinitionItemArgs } from '@sphereon/ssi-sdk.pd-manager'
+import { DeleteDefinitionArgs } from '@sphereon/ssi-sdk.data-store'
+
+export function pdReadEndpoint(router: Router, context: IRequiredContext, opts?: ISingleEndpointOpts) {
+  if (opts?.enabled === false) {
+    console.log(`"pdReadEndpoint" Endpoint is disabled`)
+    return
+  }
+  const path = opts?.path ?? '/presentation-defs'
+  router.get(`${path}/:pdId`, checkAuth(opts?.endpoint), async (request: Request, response: Response) => {
```

Comment:
is pdId the same as itemId? itemId is the db id right? pdId is the id from the pd? so in the definition?

Reply:
By now it is. fixed